formats/protobuf/src/test/java/io/cloudevents/protobuf/ProtoSupportTest.java

```diff
@@ -0,0 +1,32 @@
+package io.cloudevents.protobuf;
```

Comment:
Can you please add the copyright header?

Reply:
Ack